imp_developer.py

```diff
@@ -1669,11 +1671,12 @@ def select_existing_device(self):
         if not devices or len(devices) == 0:
             sublime.message_dialog(STR_MESSAGE_DEVICE_LIST_EMPTY)
             return
-
         # check that response has some payload
         # response should contain the list of devices
         if len(devices) > 0:
-            all_names = [(str(device["attributes"].get("mac_address")) + " - " +
+            all_names = [(
+                str("( on)" if device["attributes"].get("device_online") else "(off)") + " - " +
```

Comment:
Seems to be a lot of code duplication. For example, if I need to change the format, I have to change it in three different places.